tests/monitoring/monitoring.go

```diff
@@ -301,150 +274,6 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
-
-			Eventually(matcher.ThisMigration(migration), 2*time.Minute, 5*time.Second).Should(matcher.BeInPhase(v1.MigrationFailed), "migration creation should fail")
-
-			waitForMetricValue(virtClient, "kubevirt_migrate_vmi_scheduling_count", 0)
-			waitForMetricValueWithLabels(virtClient, "kubevirt_migrate_vmi_failed", 1, labels)
-
-			By("Deleting the VMI")
-			Expect(virtClient.VirtualMachineInstance(vmi.Namespace).Delete(context.Background(), vmi.Name, &metav1.DeleteOptions{})).To(Succeed())
-			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
-		})
-	})
-
-	Context("VM snapshot metrics", func() {
-		quantity, _ := resource.ParseQuantity("500Mi")
-
-		createSimplePVCWithRestoreLabels := func(name string) {
-			_, err := virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Create(context.Background(), &corev1.PersistentVolumeClaim{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: name,
-					Labels: map[string]string{
-						"restore.kubevirt.io/source-vm-name":      "simple-vm",
-						"restore.kubevirt.io/source-vm-namespace": util.NamespaceTestDefault,
-					},
-				},
-				Spec: corev1.PersistentVolumeClaimSpec{
-					AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
-					Resources: corev1.ResourceRequirements{
-						Requests: corev1.ResourceList{
-							"storage": quantity,
-						},
-					},
-				},
-			}, metav1.CreateOptions{})
-			Expect(err).ToNot(HaveOccurred())
-		}
-
-		It("[test_id:8639]Number of disks restored and total restored bytes metric values should be correct", func() {
-			totalMetric := fmt.Sprintf("kubevirt_vmsnapshot_disks_restored_from_source_total{vm_name='simple-vm',vm_namespace='%s'}", util.NamespaceTestDefault)
-			bytesMetric := fmt.Sprintf("kubevirt_vmsnapshot_disks_restored_from_source_bytes{vm_name='simple-vm',vm_namespace='%s'}", util.NamespaceTestDefault)
-			numPVCs := 2
-
-			for i := 1; i < numPVCs+1; i++ {
-				// Create dummy PVC that is labelled as "restored" from VM snapshot
-				createSimplePVCWithRestoreLabels(fmt.Sprintf("vmsnapshot-restored-pvc-%d", i))
-				// Metric values increases per restored disk
-				waitForMetricValue(virtClient, totalMetric, int64(i))
-				waitForMetricValue(virtClient, bytesMetric, quantity.Value()*int64(i))
-			}
-		})
-	})
-
-	Context("VM status metrics", func() {
-		newVirtualMachine := func() *v1.VirtualMachine {
-			vmi := tests.NewRandomVMI()
-			return tests.NewRandomVirtualMachine(vmi, true)
-		}
-
-		createVirtualMachine := func(vm *v1.VirtualMachine) {
-			By("Creating VirtualMachine")
-			_, err := virtClient.VirtualMachine(util.NamespaceTestDefault).Create(context.Background(), vm)
-			Expect(err).ToNot(HaveOccurred())
-		}
-
-		It("should expose VM CPU metrics", func() {
-			vm := newVirtualMachine()
-			createVirtualMachine(vm)
-
-			metrics := []string{
-				"kubevirt_vmi_cpu_system_usage_seconds",
-				"kubevirt_vmi_cpu_usage_seconds",
-				"kubevirt_vmi_cpu_user_usage_seconds",
```

Comment:
`kubevirt_vmi_cpu_system_usage_seconds` and `kubevirt_vmi_cpu_user_usage_seconds` are not checked anymore. IMHO this will reduce our coverage

Reply:
I updated the test to check the 3 metrics in each status